control-plane/api-gateway/common/translation_test.go

```diff
@@ -1428,10 +1428,10 @@ func TestTranslator_ToHTTPRoute(t *testing.T) {
 							Headers:    []api.HTTPHeaderFilter{},
 							URLRewrite: nil,
 							RetryFilter: &api.RetryFilter{
-								NumRetries:            pointer.Uint32(3),
+								NumRetries:            3,
 								RetryOn:               []string{"cancelled"},
 								RetryOnStatusCodes:    []uint32{500, 502},
-								RetryOnConnectFailure: pointer.Bool(false),
+								RetryOnConnectFailure: false,
 							},
```

Comment:
Looks like the `api` module had a breaking change. I suppose we can stick with it until someone else runs into it.

Reply:
It was technically a breaking change within a single release -- it was never released as a breaking change to users.

The consuming end was just missed until now. Still catching it before it goes out to customers